config.js

```diff
@@ -1,15 +1,26 @@
 import path from 'node:path';
-
 export const repoPath =
   process.env.MOZILLA_CENTRAL_REPO_PATH || '../mozilla-unified';
 
-const { storybookTables } = await import(
-  path.join(
-    repoPath,
-    '/toolkit/themes/shared/design-system/tokens-storybook.mjs',
-  )
+let tokensPath = path.join(
+  repoPath,
+  '/toolkit/themes/shared/design-system/tokens-storybook.mjs',
 );
 
+/*
+  tgiles: We need to prepend file:// on Windows
+  otherwise the default ESM loader will error out with
+  ERR_UNSUPPORTED_ESM_URL_SCHEME. The relevant text from
+  the error is the following: "On Windows, absolute paths
+  must be valid file:// URLs."
+*/
+
+if (process.platform === 'win32') {
```

Comment:
I wouldn't be against adding this if it's likely to be left out - but maybe it could only add a `file://` prefix if the path doesn't already start with that?